src/UseCases/ValidateAmount/ValidateAmountUseCase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\UseCases\ValidateAmount;
+
+use WMDE\Fundraising\Frontend\Validation\AmountValidator;
+use WMDE\Fundraising\Frontend\ResponseModel\ValidationResponse;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class ValidateAmountUseCase {
```

Comment:
I actually gave this exact case as an example of where to deviate to Kai... kinda facepalm now at the communication